.eslintrc

```diff
@@ -14,12 +14,16 @@
         "no-param-reassign": 0,
         "comma-dangle": 0,
         "curly": ["error", "multi-line"],
-        "prefer-destructuring": [2, {
-            "commonjs": true
-        }],
+        "prefer-destructuring": ["error", {
+            "array": true,
+            "object": true
+          }, {
+            "enforceForRenamedProperties": false
+          }],
         "import/no-unresolved": [2, {
             "commonjs": true
         }],
+        "no-plusplus" : "off",
```

Comment:
Consider turning this rule back on and use this format. A = A + 1 instead.

Reply:
It's done
![image](https://user-images.githubusercontent.com/11549858/78555414-440f9000-780d-11ea-8d4f-ed6a4dee0fa5.png)